### DISCUSSION TITLE
training a model in british english

### DISCUSSION THREAD

**Discussion by dothehansa** [coqui-ai/TTS#2368] [General]
just sharing something in case others want to train a model with british english phonemes: 

by default, choosing 'en' as a phoneme language in the models config is defaulted to 'en_us' in 'TTS/tts/utils/text/phonemizers/__init__.py line 27. 

you'll need to amend the ["en-us"] to ["en"]. 

this only affects plain British english, the regional variants & RP phonemes work with the default

(you do need to specify espeak as the phonemizer in the config or it returns a key error)

**Comment by omega3** [comment]
Have you succeeded in training British English voice? Can you share it in dedicated thread - [here](https://github.com/coqui-ai/TTS/discussions/930), please?
I have pretty week Intel G4560 with no GPU, so I have no chances at the moment to train.

**Comment by VahedCivil** [comment]
Which model is used for a British accent?
